internal/application/callback.go

```diff
@@ -37,6 +37,19 @@ func UpdateProfile(profileRequest requests.DeviceProfileRequest, dic *di.Contain
 	}
 
 	lc.Debug(fmt.Sprintf("profile %s updated", profileRequest.Profile.Name))
```

Comment:
please help update this line to use Debugf

Reply:
Updated